student/tests/factories/enrolment_factory.py

```diff
@@ -0,0 +1,33 @@
+import factory
+
+from authentication.tests.factories.student_user_factory import StudentUserFactory
+from student.models.enrolment import Enrolment
+from student.tests.factories.registration_factory import RegistrationFactory
+from staff.tests.factories.batch_factory import BatchFactory
+from staff.tests.factories.section_factory import SectionFactory
+
+class EnrolmentFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = Enrolment
+
+    registration = factory.SubFactory(RegistrationFactory, batch=factory.SelfAttribute('..batch'))
+    batch = factory.SubFactory(BatchFactory, swe_fundamentals=True)
+    section = factory.SubFactory(SectionFactory, batch=factory.SelfAttribute('..batch'))
+    student_user = factory.SubFactory(StudentUserFactory)
+    status = Enrolment.ENROLLED
+
+    class Params:
+        coding_basics = factory.Trait(
```

Comment:
Should we leave comments where there's still `coding_basics` to explain why there's both `coding_basics` and `swe_fundamentals`?